waspc/src/Wasp/AppSpec/Valid.hs

```diff
@@ -379,6 +395,44 @@ validateWebAppBaseDir spec = case maybeBaseDir of
     startsWithSlash ('/' : _) = True
     startsWithSlash _ = False
 
+validateUserNodeVersionRange :: AppSpec -> [ValidationError]
+validateUserNodeVersionRange spec =
+  concat
+    [ checkUserRangeIsInWaspRange,
+      checkUserRangeAllowsOnlyMinorChanges
+    ]
+  where
+    userRange = AS.userNodeVersionRange spec
+
+    checkUserRangeIsInWaspRange :: [ValidationError]
+    checkUserRangeIsInWaspRange =
+      if not (V.isRangeInWaspSupportedRange userRange)
+        then
+          [ GenericValidationError $
+              "Your app's Node version range ("
+                <> show userRange
+                <> ") allow versions lower than "
+                <> show oldestWaspSupportedNodeVersion
+                <> " ."
+                <> " Wasp only works with Node >= "
+                <> show oldestWaspSupportedNodeVersion
+                <> " . "
```

Comment:
Space behind and before the full stop?

Before - We agreed on it somewhere else, but I guess you forgot to drop it here.
After - Is this because we're joining the error messages?

Reply:
You are right, it went by -> fixed! Removed space before and after, both were redundant.